src/modules/handle-isDisputed.ts

```diff
@@ -0,0 +1,58 @@
+import { Network_v2 } from "@taikai/dappkit";
+import { Op } from "sequelize";
+import db from "src/db";
+import logger from "src/utils/logger-handler";
+import { updateCuratorProposalParams } from "./handle-curators";
+
+async function updateCuratorProposal(address: string, networkId: number, type: "add" | "remove") {
+  const curator = await db.curators.findOne({ where: { address, networkId }})
+  if(curator) return await updateCuratorProposalParams(curator, "disputedProposals", type)
+}
+
+export async function handleIsDisputed(
+  eventName: string,
+  networkV2: Network_v2,
+  network_id: number
+) {
+    const proposals = await db.merge_proposals.findAll({
+      where: {
+        network_id,
```

Comment:
Remove from this query proposals that was refused by bounty owner, as they can no longer be accepted